src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -153,6 +153,9 @@ public partial class Regex : System.Runtime.Serialization.ISerializable
         public System.TimeSpan MatchTimeout { get { throw null; } }
         public System.Text.RegularExpressions.RegexOptions Options { get { throw null; } }
         public bool RightToLeft { get { throw null; } }
+        public static void CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[] regexinfos, System.Reflection.AssemblyName assemblyname) { }
```

Comment:
Added by the tooling. Also this is what's causing the build to fail.

Reply:
It's conflicting with methods in the `netcoreapp` file. Could only make it compile when removing it.

Reply:
Your branch looks stale: it still has `uap` in the ref/Configurations.props file, but that's been removed in master.  At this point, the only configuration the ref builds for is netcoreapp, so there's no point in having both the RegularExpressions.cs and RegularExpression.netcoreapp.cs file.  Unless @ViktorHofer says otherwise, I suggest you just delete the .netcoreapp.cs file and leave all these methods here (ensuring that everything in the netcoreapp.cs file is in fact here now).